src/lib.rs

```diff
@@ -8,8 +8,8 @@ to the corresponding functions in `libc`.
 extern crate libc;
 
 use libc::funcs::c95::string;
-use libc::types::common::c95::c_void;
-use libc::types::os::arch::c95::{c_int, size_t};
+use libc::c_void;
+use libc::{c_int, size_t};
```

Comment:
Hah, it wasn't intentional. I always get lost in the module hierarchy of `liblibc`. :-(